src/main/frontend/components/bug_report.cljs

```diff
@@ -58,7 +58,10 @@
        (list [:div.mx-auto "Press Ctrl+V / ⌘+V to inspect your clipboard data"]
              [:div.mx-auto "or click here to paste if you are using the mobile version"]
              ;; for mobile
-             [:input {:type "text"}]))
+             [:input.form-input.is-large.transition.duration-150.ease-in-out {:type "text" :placeholder "Long press here to paste if you are on mobile"}]
+             [:div.flex.justify-between.items-center.mt-2
+              [:div "Something wrong? No problem, click here to go to the previous step."]
+              (ui/button "Go back" :on-click #(util/open-url (rfe/href :bug-report)))]))
```

Comment:
This `:on-click` pointed here will mess up the history stack since it pushes the href to the stack instead of popping, as the following screen recording shows.

https://user-images.githubusercontent.com/28241963/212530055-97f37766-76f4-4e52-9822-41867cb6edc3.mp4